### PR TITLE
Adding Host to response headers

### PR DESCRIPTION
**Pull request by q4chrisj**
The response headers will now contain a field "Host:domain"

Signed-off-by: Chris Jones <chrisj@q4inc.com>

### CODE REVIEW

File: proxy.conf [hunk 1]
```diff
@@ -12,4 +12,5 @@ proxy_cache_bypass      $arg_nocache;
 # proxy_ssl_session_reuse off is required to work with Barracuda
 proxy_ssl_session_reuse off;
 proxy_cache_methods     GET HEAD;
-add_header              X-Cache-status $upstream_cache_status always;
\ No newline at end of file
+add_header              X-Cache-status $upstream_cache_status always;
+add_header              Host $Host always;
```
Comment (maximv-q4) [inline proxy.conf]: why are we doing this?

**Comment by q4chrisj** [comment]
@maximv-q4 we have a ticket in our queue to add this: https://q4websystems.atlassian.net/browse/DEVOPS-4144

File: proxy.conf (hunk 1 above)
Comment (q4chrisj) [reply proxy.conf]: we have a ticket to add this: https://q4websystems.atlassian.net/browse/DEVOPS-4144

Comment (q4chrisj) [reply proxy.conf]: @maximv-q4  ok added the $hostname (which is the machine name) based on our discussion

**Pull request merged**